openmetadata-spec/src/main/resources/json/schema/entity/teams/user.json

```diff
@@ -155,6 +155,14 @@
     "lastLoginTime": {
       "description": "Last time the user logged in.",
       "$ref": "../../type/basic.json#/definitions/timestamp"
+    },
```

Comment:
IMO we should use some other table for user preferences .
We have "personaPreference" as one type , bad example maybe appearance and color choice can be as per user choice.

Reply:
just want this to be in a way where it's flexible enought to accomodate other type of per user preferences